test/unit/ui/marker.test.js

```diff
@@ -17,62 +17,201 @@ function createMap() {
 
 test('Marker', (t) => {
     t.test('constructor', (t) => {
-        const el = window.document.createElement('div');
+        const el = window.document.createElement(`mapboxgl-marker`);
```

Comment:
this will try to make an Element like `<mapboxgl-marker>` let's stick with a `<div>` element.